index.js

```diff
@@ -0,0 +1,74 @@
+import Book from './modules/constructor.js';
+import UI from './modules/ui.js';
+import Store from './modules/store.js';
+
+// eslint-disable-next-line max-classes-per-file
+const { luxon } = window;
+const dt = luxon.DateTime.now();
```

Comment:
- [ ] As it is a requirement for this project, you are suppose to display the date using `luxon` library. Please check the screenshot of the requirements below and kindly fix that.
![luxon2](https://user-images.githubusercontent.com/65852621/149945930-14db171f-a757-4cd4-85f3-f1ea3a0a66c9.png)